pkg/migrations/op_drop_constraint.go

```diff
@@ -96,6 +103,18 @@ func (o *OpDropConstraint) Complete(ctx context.Context, conn db.DB, tr SQLTrans
 }
 
 func (o *OpDropConstraint) Rollback(ctx context.Context, conn db.DB, tr SQLTransformer) error {
+	// TODO: We need access to the schema here so that we can look up the column name from the constraint
+	// name. We have a few options:
+	//
+	// 1. Update the signature of this method to accept a schema object like the others. This would force
+	// us to change our Operation interface.
+	//
+	// 2. Fetch the schema manually from the db using a query and unmarshal it. This feel like a bit of a
+	// specific hack but would save us from changing the Operation interface.
+	//
+	// 3. An even bigger hack, embed the schema in the context for this case only. I don't like it, but
+	// wanted to mention it as an option.
+
```

Comment:
@exekias @andrew-farries WDYT? 

Maybe changing the interface to include the schema is the best option (Option 1). Just want some confirmation since maybe there was a reason for omitting it up to this point.

Reply:
I went with updating the interface as it felt the most consistent with the rest of the code.